ci/containers_build_push.sh

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
+
+# This script is not intended for humans.  It should be run by secure
+# (maintainer-only) cron-like automation or in maintainer-authorized PRs.
+# Its primary purpose is to build and push multi-arch skopeo, buildah, podman
+# container images to multiple locations.  The destination repository namespace
+# as well as the image contents are controlled by the "FLAVOR" argument
+# described below.
+#
+# The first argument to the script, should be the git URL of the repository
+# containing the build context.  This is assumed to be the $CWD. This URL will
+# be used to add several labels the images identifying the context source.
+#
+# The second argument to this script is the path (relative to the first argument)
+# of the build context subdirectory.  In other words, the subdirectory holding
+# the `Containerfile`.  If no `Containerfile` is found, the build will fail.
+#
+# The third argument indicates the image "FLAVOR", which will be passed into
+# the build as a `--build-arg`.  This is used by the `Containerfile` to alter
+# the build to produce a 'stable', 'testing', or 'upstream' image.  Importantly,
+# this also determines where the image is pushed, see the top-level README.md for
+# more details.
+#
+# Optionally, the `$ARCHES` environment variable may be set to a comma-separated
+# list of golang-centric architectures to include in the build.  It is assumed
+# that the necessary emulation is setup to handle building of non-native arches.
+# Note: these builds will run in parallel, which can make the output difficult
+# to read.
```

Comment:
Great description here, but no metnion of immutable, it might be nice to add.

Reply:
good catch, I'll mention it.